resource-service/README.md

```diff
@@ -50,9 +50,14 @@ The service accepts a `ResourceRequest` from the User Service, finds all the res
 
```

Comment:
Stiil need to address these two points:

- Update the architectural diagram to show going to an actual source of resource metadata info. I like the user service way of showing this
- Match terminology, we refer to backing store in the intro text but the backing store isnt in the diagram

Reply:
These have been addressed